modules/mag-deck/mag-deck-arduino/mag-deck-arduino.ino

```diff
@@ -45,7 +49,7 @@ Memory memory = Memory();  // reads from EEPROM to find device's unique serial,
 
 #define CURRENT_TO_BYTES_FACTOR 114
 
-#define STEPS_PER_MM 50  // full-stepping
+#define STEPS_PER_MM ((model_version < 20) ? 50 : 100)  // full-stepping
```

Comment:
Not a big fan of this approach, though it would work fine. It checks the value every single time it's called, and putting a ternary in a non-parameterized macro like this hides that there's logic involved.

Instead, let's make a new struct that pulls together all the motion params that we adjust here; let's instantiate one on the free store; and let's adjust its values depending on the model and check them there. This will also pull it all together nicely. Call it like MotionParams or something.